doc/extension.rst

```diff
@@ -61,11 +61,18 @@ The extension has a few configuration options that can be set in ``conf.py``:
       are likely to change.
 
 .. py:data:: cautodoc_clang
-   :type: str
+   :type: list, str
+
+   A list of arguments to pass to ``clang`` while parsing the source, typically
+   to add directories to include file search path, or to define macros for
+   conditional compilation. May be specified as a Python list, or as a string
+   with arguments separated by commas. No arguments are passed by default.
```

Comment:
Come to think of it `subprocess.run` needs a list of individual arguments, so what I say above is slightly off the mark. However splitting the parameter string on whitespace instead of commas would be more robust and then match the behaviour of the list more closely.

Reply:
Trouble is, it's *currently* a comma-separated list. This is merely retaining backward compatibility while adding support for actual Python lists.

Reply:
Ah, I see. Do we care about backwards compatibility at this point though? Anyway, that's a valid counterargument otherwise, so it's up to you I think.

Reply:
By "at this point" I mean in versions <1.0, not that we've been breaking it so far :sweat_smile: 

Resolving the conversation by the way as this is a trivial issue whichever way you choose to deal with it.

Reply:
Ugh, it seems 'resolving' just hid it. Anyway, I'm not sure you get notifications otherwise so I'm 'unhiding' it just in case, sorry for the spam.